### PR TITLE
Don't load the pixel, cell, and neighborhood FOVs in batches for mask generation

### LINKED ISSUE #815: Don't load in pixel and cell masks in batch
**Describe the bug**

Currently, if the user has images of different sizes, the pixel and cell masks don't load in properly. This is because it's still using the legacy `load_imgs_from_dir` function.

**Expected behavior**

The masks should load and display properly.

**To Reproduce**

Run the notebook as is with a cohort of images with varying sizes.

### PR DESCRIPTION
**Pull request by alex-l-kong**
**What is the purpose of this PR?**

Closes #815. Resolves an issue with variable-sized images using the legacy batch loading technique for mask generation.

**How did you implement your changes**

Implement a `for` loop to generate these instead of using one `load_imgs_from_dir` call.

### CODE REVIEW

File: templates/2_Cluster_Pixels.ipynb [hunk 1]
```diff
@@ -793,21 +793,22 @@
    "source": [
```
Comment (ngreenwald) [inline templates/2_Cluster_Pixels.ipynb:793]: <pre class="ql-syntax" spellcheck="false">Line #3.    for pixel_fov in subset_pixel_fovs:
</pre><p>What's the point of the subset_pixel_fovs variable?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/ark-analysis/pull/816/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='51'/>

**Review by ngreenwald — CHANGES_REQUESTED** [review]
Looks good, one notebook cleanup

File: templates/2_Cluster_Pixels.ipynb (hunk 1 above)
Comment (alex-l-kong) [reply templates/2_Cluster_Pixels.ipynb:793]: <p>Looks like an artifact from a previous update, I'll remove this.</p>

Comment (alex-l-kong) [reply templates/2_Cluster_Pixels.ipynb:793]: <p>Actually, scratch that. I'll remove <code>pixel_fovs</code> and just use <code>subset_pixel_fovs</code>. More intuitive to name it that way.  </p>

**Comment by alex-l-kong** [comment]
@ngreenwald

**Review by ngreenwald — APPROVED** [review]
Looks good

**Pull request merged**